composer.json

```diff
@@ -55,5 +56,15 @@
         "bin-dir": "inc/vendor/bin",
         "optimize-autoloader": true,
         "sort-packages": true
+    },
+    "scripts": {
+        "phpstan": "phpstan analyse src/ -c inc/src/phpstan.neon --level=0 --no-progress -vvv",
```

Comment:
It looks like using the same name for these scripts causes some issues:

```
Key phpstan is a duplicate in ./composer.json at line 62
```

Might be best to make them `phpstan-0`, `phpstan-1`, etc.